src/screens/PrivateChat/ContactsPicker/index.js

```diff
@@ -0,0 +1,259 @@
+// @flow
+
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  View,
+  ScrollView,
+  TouchableOpacity,
+  Clipboard,
+  Image,
+} from 'react-native';
+import { Text } from 'native-base';
+import { getProfile, newChatSession } from '../../../actions/chat';
+import { addContact } from '../../../actions/contacts';
+import BackgroundImage from '../../../components/common/BackgroundImage';
+import styles from './styles';
+import FakeNavigationBar from '../../../components/common/FakeNavigationBar';
+import Loading from '../../../components/common/Loading';
+import NavigatorComponent from '../../../components/common/NavigatorComponent';
+import i18n from '../../../global/i18n';
+import Colors from '../../../global/colors';
+import type { ProfileType } from '../../../types/Chat';
+import ScreenTitle from '../../../components/common/ScreenTitle';
+import InvalidKeyModal from './InvalidKeyModal';
+import AssetsImage from '../../../global/AssetsImages';
+import { imageSource } from '../../../utils/profile';
+
+import { Chip, Selectize } from 'react-native-material-selectize';
+
+const DONE_BUTTON = 'DONE_BUTTON';
+const INVALID_MODAL_KEY = 'invalidKey';
+
+type Props = {
```

Comment:
We should be able to pass initially selected contacts to the picker via props, so we can implement editing of members list.